### ISSUE TITLE
Install of fake on Ubuntu succeeds but command is not found

### ISSUE THREAD

**Issue by halcwb** [fsharp/FAKE#2091]
### Description

With both dotnet core and mono installed on Ubuntu 18.04, I installed fake globally using the dotnet tool install command. I then restart the terminal to test if fake is working with fake --version

### Repro steps

See above

### Expected behavior

Getting the version of fake

### Actual behavior

Command 'fake' not found

### Known workarounds

I probably need to still add a path to fake. 

### Related information 

* Ubuntu 18.04
* Branch
* donet core 2.1.4.02
* Indications of severity
* fake 5 cli
* (if performance issue) Performance information, links to performance testing scripts

**Comment by baronfel** [comment]
What's your shell? There's a known issue with zsh/fish where the default tool install location isn't correctly added to the PATH: https://github.com/dotnet/cli/issues/9321

**Comment by halcwb** [comment]
Yep I had to add to my path, I think I am using zsh. So for documentation I added:
`export PATH=$PATH:/home/halcwb/.dotnet/tools/` to the `~/.bashrc` file. And then it works.

**Comment by matthid** [comment]
Please send a PR to update the documentation

**Comment by kristianmandrup** [comment]
Sweet :)
